meta-facebook/minerva-ag/src/platform/plat_pldm_fw_update.c

```diff
@@ -573,10 +573,11 @@ static bool get_vr_fw_version(void *info_p, uint8_t *buf, uint8_t *len)
 	memcpy(buf_p, vr_name_p, strlen(vr_name_p));
 	buf_p += strlen(vr_name_p);
 
-	if (sensor_dev == sensor_dev_mp2891 || sensor_dev == sensor_dev_mp29816a)
+	if (sensor_dev == sensor_dev_mp2891 || sensor_dev == sensor_dev_mp29816a) {
 		*len += bin2hex((uint8_t *)&version, 2, buf_p, 4) + strlen(vr_name_p);
-	else if (sensor_dev == sensor_dev_isl69259 || sensor_dev == sensor_dev_raa228238 ||
-		 sensor_dev == sensor_dev_raa228249 || sensor_dev == sensor_dev_mp2971)
+		*len += 4;
```

Comment:
@LisaChang-Quanta I am merging this PR, but can you please clarify why you are doing +4 here?